app/views/transferComplete.scala.html

```diff
@@ -4,7 +4,7 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
-        @progressIndicator(Messages("transferComplete.progress"))
```

Comment:
Just to check - are you and Kirsten both happy with the fact that this page won't have a Step header on it at all? I'd have thought users would want to see what step they're on.

Reply:
Ah, sorry, this is the final page, isn't it? So I guess it doesn't make sense to have a Step label here.

Reply:
Yes it is the final page, I will double-check with Kirsten though! Thank you!